src/analysis/ReachingDefinitions/ReachingDefinitions.h

```diff
@@ -115,13 +115,22 @@ class RDNode : public SubgraphNode<RDNode> {
         uses.insert(std::forward<T>(ds));
     }
 
-    void addUses(std::vector<DefSite>& u)
+    template <typename T>
+    void addUses(T&& u)
     {
-        for (DefSite& ds : u) {
+        for (auto ds : u) {
```

Comment:
shouldn't here be `auto&` to avoid useless copying? What is the type inside `u`?